foundry/mouseInteractionManager.d.ts

```diff
@@ -1,213 +1,252 @@
 /**
  * Handle mouse interaction events for a Canvas object.
- *
  * There are three phases of events: hover, click, and drag
  *
  * Hover Events:
- *
- *      _handleMouseOver
- *              action: hoverIn
- *      _handleMouseOut
- *              action: hoverOut
+ * _handleMouseOver
+ *  action: hoverIn
+ * _handleMouseOut
+ *  action: hoverOut
  *
  * Left Click and Double-Click
- *
- *      _handleMouseDown
- *              action: clickLeft
- *              action: clickLeft2
+ * _handleMouseDown
+ *  action: clickLeft
+ *  action: clickLeft2
  *
  * Right Click and Double-Click
- *
- *      _handleRightDown
- *              action: clickRight
- *              action: clickRight2
+ * _handleRightDown
+ *  action: clickRight
+ *  action: clickRight2
  *
  * Drag and Drop
- *
- *      _handleMouseMove
- *              action: dragLeftStart
- *              action: dragLeftMove
- *              action: dragRightStart
- *              action: dragLeftMove
- *      _handleMouseUp
- *              action: dragLeftDrop
- *              action: dragRightDrop
- *      _handleDragCancel
- *              action: dragLeftCancel
- *              action: dragRightCancel
+ * _handleMouseMove
+ *  action: dragLeftStart
+ *  action: dragLeftMove
+ *  action: dragRightStart
+ *  action: dragLeftMove
+ * _handleMouseUp
+ *  action: dragLeftDrop
+ *  action: dragRightDrop
+ * _handleDragCancel
+ *  action: dragLeftCancel
+ *  action: dragRightCancel
```

Comment:
Did you want to align them to the start of the word, or why the indenting changes here?

Reply:
I'll double check, but I believe the indentation changed in foundry itself.

Reply:
Yeah, the indenting changed in foundry.js. I merely copied over the new jsdocs.